src/components/adherence/participant-detail/EditParticipantNotes.tsx

```diff
@@ -0,0 +1,98 @@
+import {makeStyles, Paper, TextField, Button} from '@material-ui/core'
+import {MTBHeadingH4} from '@components/widgets/Headings'
+import ParticipantService from '@services/participants.service'
+import { useUserSessionDataState } from '@helpers/AuthContext' 
+import {ExtendedError, ParticipantAccountSummary, EnrolledAccountRecord} from '@typedefs/types'
+import React, {FunctionComponent} from 'react'
+
+type EditParticipantNotesProps = {
+    participantId : string,
+    studyId: string,
+    enrollment: EnrolledAccountRecord
+}
+
+const EditParticipantNotes: FunctionComponent<EditParticipantNotesProps> = ({
+    participantId,
+    studyId,
+    enrollment
+}) => {
+    const useStyles = makeStyles(theme => ({
+        notesContainer: {
+          marginTop: theme.spacing(8),
+          padding: theme.spacing(4)
+        },
+        notesText: {
+            "& .MuiOutlinedInput-inputMultiline": {
+                background: '#fff',
+                padding: theme.spacing(1),
+            },
+            "& .MuiOutlinedInput-notchedOutline": {
+                borderColor: '#BBC3CD',
+            },
+            marginTop: theme.spacing(2),
+            marginBottom: theme.spacing(2),
+            // border:'1px solid #BBC3CD',
+            width: '55%',
+        },      
+        textArea: {
+            backgroundColor: '#f8f8f8',
+        },
+        saveBtn: {
```

Comment:
I think we have a custom <SaveButton/> element that matches the style closely enough